README.md

```diff
@@ -38,7 +38,7 @@ Launches the test runner in the interactive watch mode also running the [jest-ax
 
 Lunches the test runner with coverage
 
-### `make build`
+### `make create-build`
```

Comment:
suggestion: update the makefile instead. The backend used `make build`.

Reply:
For the web client the command is `make build-docker`

Reply:
@josebui that is the command to build the docker image, there is also a `create-build` command in the Makefile which creates a production build of the web app